pkg/network/protocols/http/model_windows.go

```diff
@@ -106,14 +106,37 @@ func (tx *WinHttpTransaction) StaticTags() uint64 {
 //
 //nolint:revive // TODO(WKIT) Fix revive linter
 func (tx *WinHttpTransaction) DynamicTags() []string {
-	if len(tx.AppPool) != 0 || len(tx.SiteName) != 0 {
-		return []string{
-			fmt.Sprintf("http.iis.app_pool:%v", tx.AppPool),
-			fmt.Sprintf("http.iis.site:%v", tx.SiteID),
-			fmt.Sprintf("http.iis.sitename:%v", tx.SiteName),
-		}
+	tags := make([]string, 0, 6)
+	tags = append(tags, fmt.Sprintf("http.iis.site:%v", tx.SiteID))
+	if (len(tx.AppPool)) > 0 {
+		tags = append(tags, fmt.Sprintf("http.iis.app_pool:%v", tx.AppPool))
 	}
-	return nil
+	if (len(tx.SiteName)) > 0 {
+		tags = append(tags, fmt.Sprintf("http.iis.sitename:%v", tx.SiteName))
+	}
+
+	// tag precedence is web.config -> datadog.json
```

Comment:
I hope this detail doesn't get lost.

Reply:
The internal documentation has the full description of the precedence.